lyricsbot/tests/domains/lyricsondemand/test_lyricsondemand.py

```diff
@@ -0,0 +1,120 @@
+"""
+Tests.
+"""
+import unittest
+
+from ddt import ddt, data, unpack
+
+from lyricsbot.domains.lyricsondemand import (
+    format_request_data_url,
+    parse_lyrics
+)
+
+
+@ddt
+class TestURL(unittest.TestCase):
+    """
+    Test for verify data song with url and parser.
+    """
+
+    @data(
+        (
+            'Childish Gambino',
+            'Redbone',
+            'https://www.lyricsondemand.com/c/childishgambinolyrics/redbonelyrics.html',
+        ),
+        (
+            'Florence + The Machine',
+            'Rabbit Heart (raise it up)',
+            'https://www.lyricsondemand.com/f/florenceandthemachinelyrics/rabbitheartraiseituplyrics.html',
+        ),
+    )
+    @unpack
+    def test_format_request_data_url(self, author_song, title_song, expected_url):
+        """
+        Test for verify the URL format.
```

Comment:
If you have `case` and `expected` point, you do not need additional doc-string.

Reply:
Done.